### ISSUE TITLE
Create GUI Template

### ISSUE THREAD

**Issue by Yunehr** [Yunehr/Submarine-SCADA-HMI_Project-3_Group15#59]
Design and implement the Template/Main Page GUI where alt text is placed, where other implementations should be
